### DISCUSSION TITLE
请问有无便捷方式添加上标或者下标

### DISCUSSION THREAD

**Discussion by ganbeiqisi** [obsidianzh/forum#118] [General综合]
如题，有时候希望添加字母上下标，但是找了半天没这个功能。

**Comment by chuanwenzhao** [comment]
安装一个cmenu的插件就好了
![image](https://user-images.githubusercontent.com/42823366/133371976-4c5ba522-1032-418d-82a7-ec9d63f50cf0.png)

**Comment by Lillian229** [comment]
你好，请问这些小图标是怎么实现的啊
